openquake/hazardlib/gsim/abrahamson_gulerce_2020.py

```diff
@@ -711,6 +755,7 @@ class AbrahamsonGulerce2020SSlab(AbrahamsonGulerce2020SInter):
     """
     #: Required rupture parameters are magnitude and top-of-rupture depth
     REQUIRES_RUPTURE_PARAMETERS = {'mag', 'ztor'}
+    REQUIRES_SITES_PARAMETERS = {'vs30', 'backarc'}
```

Comment:
Rather than modifying the original AG20 I think it would have been better to create a separate class containing the backarc term correction. With this change we loose back compatibility since we are making mandatory a site parameter not requested in the original version.